rules.json

```diff
@@ -5,23 +5,15 @@
     "type": "ResourceGraph",
     "query": "Resources | where type=~ 'Microsoft.Storage/storageAccounts' and isnotempty(properties.privateEndpointConnections[0]) and properties.networkAcls.defaultAction=~ 'Allow'"
   },
-  {
-    "name": "event-hubs-default-action-deny",
-    "description": "Finds Event Hubs with a network rules set having one or more IP or virtual network rules but the default action is not deny",
-    "type": "ARM",
-    "evaluation": {
-      "jmespath": "type == `Microsoft.EventHub/namespaces/networkRuleSets` && properties.defaultAction == `Deny` && length(properties.ipRules) == 0 && length(properties.virtualNetworkRules) == 0"
-    }
-  },
   {
```

Comment:
I expected to see use of the new `recommendation` property here in the rules. I didn't see a fallback/default that pointed to `built-in-rules.md`. How will users know how to resolve issues?